easybuild/tools/github.py

```diff
@@ -1117,8 +1182,10 @@ def _easyconfigs_pr_common(paths, ecs, start_branch=None, pr_branch=None, start_
 
     diff_stat = git_repo.git.diff(cached=True, stat=True)
     if not diff_stat:
-        raise EasyBuildError("No changed files found when comparing to current develop branch. "
-                             "Refused to make empty pull request.")
+        raise EasyBuildError(
+            "No changed files found when comparing to current develop branch. Refused to make empty pull request.",
```

Comment:
While we're at it: `develop` shouldn't be hardcoded here:
```suggestion
            f"No changed files found when comparing to current {start_branch} branch. "
            "Refused to make empty pull request.",
```